src/gam.py

```diff
@@ -11051,6 +11050,16 @@ def sortCSVTitles(firstTitle, titles):
   for title in restoreTitles[::-1]:
     titles.insert(0, title)
 
+def QuotedArgumentList(items):
```

Comment:
- we need to escape the text also in case the user ran something like:

gam print users query "abc=\"def\""

- instead of qstr and needing to remove the last space, make a qarray and return the join.

Reply:
You got lucky with your sample query; it's the same with and without the quotes.
$ gam print users query "name="TestUser1""
Getting all Users in G Suite account that match query (name=TestUser1) (may take some time on a large account)...
Got 0 Users:  - 
primaryEmail
$ gam print users query name=TestUser1  
Getting all Users in G Suite account that match query (name=TestUser1) (may take some time on a large account)...
Got 0 Users:  - 
primaryEmail

As soon as you really need quotes it breaks:
$ gam print users query "name="Test User1""

ERROR: User1 is not a valid argument for "gam print users"

This doesn't work because the API wants single quotes:
$ gam print users query name="Test User1" 
Getting all Users in G Suite account that match query (name=Test User1) (may take some time on a large account)...
Got 0 Users:  - 
primaryEmail

This doesn't work either because the single quotes get stripped.
$ gam print users query name='Test User1'
Getting all Users in G Suite account that match query (name=Test User1) (may take some time on a large account)...
Got 0 Users:  - 
primaryEmail

This is the valid way to express the query:
$ gam print users query "name='Test User1'"
Getting all Users in G Suite account that match query (name='Test User1') (may take some time on a large account)...
Got 2 Users: testuser1@rdschool.net - testuser1@rdschool.org
primaryEmail
testuser1@rdschool.net
testuser1@rdschool.org

I've thought about this and haven't thought of anything better than surrounding any argument with a space or comma in it with double quotes.